eng/pipelines/typespec-ci.yml

```diff
@@ -14,6 +14,11 @@ pr:
     - package-lock.json
     - eng/pipelines/typespec-ci.yml
     - eng/pipelines/templates/steps/typespec-ci.yml
+    - specification/contosowidgetmanager
```

Comment:
@mikeharder what is our longer term goal here? Do we intend to keep this manually updated or do we plan to let this run for all spec PRs and essentially noop if there are no TypeSpec changes detected?

Reply:
OK I'm good for now with this approach. I would like to avoid the one-pipeline per spec approach as I think that is going to cause more issues than it solves and I want to dial that back in our language repos overtime.

Reply:
> OK I'm good for now with this approach. I would like to avoid the one-pipeline per spec approach as I think that is going to cause more issues than it solves and I want to dial that back in our language repos overtime.

I'd like to discuss this further, since I think "one-pipeline-per-[spec|sdk]" works better than "one pipeline per check which runs on all PRs and no-ops if not relevant".